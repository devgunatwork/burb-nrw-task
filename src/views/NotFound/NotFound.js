import React from 'react';
import Typography from '@material-ui/core/Typography';
import StyledLink from '../../components/StyledLink';

import styles from './NotFound.module.scss';

const NotFound = () => (
  <div className={styles.root}>
    <Typography variant="h1">404 - Not Found!</Typography>
    <Typography variant="h4">
      <StyledLink to="/">Go Home</StyledLink> <br />
    </Typography>
  </div>
);

export default NotFound;
