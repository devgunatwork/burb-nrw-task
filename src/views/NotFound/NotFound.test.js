import React from 'react';
import NotFound from './NotFound';

describe('views', () => {
  describe('NotFound', () => {
    describe('UI', () => {
      it('should match the snapshot', () => {
        const wrapper = shallow(<NotFound />);

        expect(wrapper).toMatchSnapshot();
      });
    });
  });
});
