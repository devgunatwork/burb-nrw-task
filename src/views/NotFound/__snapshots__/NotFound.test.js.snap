// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`views NotFound UI should match the snapshot 1`] = `
<div>
  <h1>
    404 - Not Found!
  </h1>
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
